docs/EVG.md

```diff
@@ -1,39 +1,35 @@
 
 The primary purposes of Extended Validation Certificates are to:  1) identify the legal entity that controls a Web or service site, and 2) enable encrypted communications with that site.  The secondary purposes include significantly enhancing cybersecurity by helping establish the legitimacy of an organization claiming to operate a Web site, and providing a vehicle that can be used to assist in addressing problems related to distributing malware, phishing, identity theft, and diverse forms of online fraud.
 
-## Notice to Readers
+**Notice to Readers**
 
-The Guidelines for the Issuance and Management of Extended Validation Certificates present criteria established by the CA/Browser Forum for use by certification authorities when issuing, maintaining, and revoking certain digital certificates for use in Internet Web site commerce.  These Guidelines may be revised from time to time, as appropriate, in accordance with procedures adopted by the CA/Browser Forum.  Questions or suggestions concerning these guidelines may be directed to the CA/Browser Forum at questions@cabforum.org.
+The Guidelines for the Issuance and Management of Extended Validation Certificates present criteria established by the CA/Browser Forum for use by certification authorities when issuing, maintaining, and revoking certain digital certificates for use in Internet Web site commerce.  These Guidelines may be revised from time to time, as appropriate, in accordance with procedures adopted by the CA/Browser Forum.  Questions or suggestions concerning these guidelines may be directed to the CA/Browser Forum at <questions@cabforum.org>.
 
-## The CA/Browser Forum
+**The CA/Browser Forum**
 
-The CA/Browser Forum is a voluntary open organization of certification authorities and suppliers of Internet browsers and other relying-party software applications.  Membership is listed at https://cabforum.org/members/.
+The CA/Browser Forum is a voluntary open organization of certification authorities and suppliers of Internet browsers and other relying-party software applications.  Membership is listed at <https://cabforum.org/members/>.
 
 ## Document History
 
-| Ver. | Ballot | Description | Adopted | Effective\* |
-| --- | --- | --- | --- | --- |
+| **Ver.** | **Ballot** | **Description** | **Adopted** | **Effective\*** |
```

Comment:
This aligns with the BRs. We can of course see how it looks changed.

Reply:
Is it worth removing white spaces on this take? I understand that you've done so much already, perhaps I could take a stab at it and remove double spaces in various places and commit the changes on this PR. What do you think?

Reply:
To avoid ambiguity: please do **not** directly edit this PR. Please use a branch of this and submit a PR to this branch so we can discuss. I mention it because in the past you’ve directly edited, and that’s not ideal.

It seems you’re raising a separate issue than what we’re discussing here, so it’s usually better to start a new thread, but you can see I intentionally did not try to align all of the EVG sentence spacing, because that would have changed nearly all of the EVG. I would rather do that as a follow-up, where we can use a redline diff of the output to show nothing functional changes (as the double spaces are ignored when doing layout to PDF or HTML)